tock/hours/forms.py

```diff
@@ -101,7 +110,7 @@ def clean(self):
             # print(form)
             if form.cleaned_data:
                 # Easy way of telling if we have the right data
-                if form.cleaned_data.get('hours_spent') == None:
+                if not form.cleaned_data.get('hours_spent'):
```

Comment:
I think that hours spent should not be blank or 0: neither of these values add useful information. I'll write some tests to make this more explicit.